src/components/select/simple-select/simple-select.component.js

```diff
@@ -475,6 +477,12 @@ SimpleSelect.propTypes = {
   listPlacement: PropTypes.oneOf(["top", "bottom", "right", "left"]),
   /** Use the opposite list placement if the set placement does not fit */
   flipEnabled: PropTypes.bool,
+  /** Set this prop to enable a virtualised list of options. It corresponds to the number of options to render into
+   * the DOM at once, either side of the currently-visible ones.
+   * Higher values make for smoother scrolling but may impact performance.
+   * If this prop is not used then all options will be in the DOM at all times, which may cause performance problems
+   * on very large lists */
+  virtualScrollBuffer: PropTypes.number,
```

Comment:
To be honest I think we should have another prop for enabling virtualScroll, like `virtualScrollEnabled`. `virtualScrollBuffer` should stay as an optional prop though.

Reply:
Fair point on the naming - I'm not very good at naming myself! I was thinking overscan might tie it too closely to this one library in case we change it later (I'd still like to move to React Aria in the longer term), but if that's a common term across several libraries it's less of an issue.

Happy to debate having a second prop and having a default for the overscan. I'm not really sure what's best for consumers. I do think they should have the ability to control it though, as I have no confidence in being able to pick a sensible default that works well for all usecases and all devices!

Reply:
I think it might be confusing for the users that they have to figure out what `overscan` value they have to provide, that's why I think we should provide a default value for that.

I'd name this prop `virtualScrollOverscan` and default it to 3 elements, but it seems like it will break the current implementation where you default the `virtualScrollBuffer` to `React.Children.count(children)`